src/app/api/services/dot-content/dot-content.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
```

Comment:
This can't be named content service because what is doing is triggering workflows actions, please name accordingly.